### PR TITLE
Improve code style of Proxy handler example

### PR DESCRIPTION
**Pull request by sentyaev**
#### Summary

Fixes missing colon and `function` keyword in [A complete traps list example](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy#a_complete_traps_list_example) code sample on Proxy page.

#### Motivation

Fix typo.

#### Supporting details

--
#### Related issues

--
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Hi—this code is correct, and we prefer the method syntax these days. Could you refactor the other code to this syntax instead?

**Comment by sentyaev** [comment]
> Hi—this code is correct, and we prefer the method syntax these days. Could you refactor the other code to this syntax instead?

Sure! np
@Josh-Cena I'm thinking to also make variable names in this code block more consistent with the rest of this file code samples. This code block has Hungarian notation for variable name, so I'm going to change it to be JS'y names.

**Comment by teoli2003** [comment]
Yes, please, feel free to remove variables named using Hungarian notation.

File: files/en-us/web/javascript/reference/global_objects/proxy/index.md [hunk 1]
```diff
@@ -395,31 +395,31 @@ Now in order to create a complete sample `traps` list, for didactic purposes, we
...
-  has: function (oTarget, sKey) {
-    return sKey in oTarget || oTarget.hasItem(sKey);
+  has(target, key) {
+    return key in target || target.hasItem(key);
   },
-  defineProperty: function (oTarget, sKey, oDesc) {
-    if (oDesc && 'value' in oDesc) { oTarget.setItem(sKey, oDesc.value); }
-    return oTarget;
+  defineProperty(target, key, descriptor) {
+    if (descriptor && 'value' in descriptor) { target.setItem(key, descriptor.value); }
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/proxy/index.md:416]:
```suggestion
    if (descriptor && 'value' in descriptor) {
      target.setItem(key, descriptor.value);
    }
```

**Comment by sentyaev** [comment]
@Josh-Cena I think it's done. Ping me in case you thing more changes needed.

**Review by Josh-Cena — APPROVED** [review]
Thanks, the change looks good to me.

Congratulations on your first contribution here—welcome aboard!

**Pull request merged**